test/ussd_nurse.test.js

```diff
@@ -259,23 +266,15 @@ describe("app", function() {
                             , {session_event: 'new'}  // redial
                         )
                         .check(function(api) {
-                            var smses = _.where(api.outbound.store, {
-                                endpoint: 'sms'
-                            });
-                            var sms = smses[0];
-                            assert.equal(smses.length,1);
-                            assert.equal(sms.content,
-                                "Please dial back in to *120*550*5# to complete the NurseConnect registration."
-                            );
-                            assert.equal(sms.to_addr,'27821234444');
+                            utils.check_fixtures_used(api, [150, 160, 163]);
```

Comment:
This test and the test below actually show that the redial sms is not being sent since no outbound message fixture is being used. Please skip these tests and add a todo in the code for sending dialback smss that references https://github.com/praekeltfoundation/ndoh-jsbox/issues/49.